Submitted By: BLFS Book <blfs-book@linuxfromscratch.org>
Date: 2004-02-01
Initial Package Version: 2.1.5
Origin: Freetype FAQ
Description: Enables bytecode interpreter for TT driver.

$LastChangedBy$
$Date$

diff -ur freetype-2.1.5.orig/include/freetype/config/ftoption.h freetype-2.1.5/include/freetype/config/ftoption.h
--- freetype-2.1.5.orig/include/freetype/config/ftoption.h	2004-02-01 02:29:17.000000000 -0600
+++ freetype-2.1.5/include/freetype/config/ftoption.h	2004-02-01 02:31:42.000000000 -0600
@@ -399,7 +399,7 @@
   /*   Do not #undef this macro here, since the build system might         */
   /*   define it for certain configurations only.                          */
   /*                                                                       */
-/* #define  TT_CONFIG_OPTION_BYTECODE_INTERPRETER */
+#define  TT_CONFIG_OPTION_BYTECODE_INTERPRETER
 
 
   /*************************************************************************/
